src/js/test/index.test.js

```diff
@@ -72,6 +76,40 @@ describe('setupHeader', () => {
         // Act
         TestUtils.click(button);
 
+        // Assert
+        const html = document.documentElement;
+        expect(html.className).toEqual('is-navInView');
+    });
+
+    it('adds `is-navInView--noPad` class to html element if header has `c-header--transparent` class', () => {
```

Comment:
Should this read `is-navInView` and `is-navInView--noPad`?

Reply:
The test here is just to test that the `is-navInView--noPad` is added (there's a separate test for `is-navInView` being added as that should get added no matter what the state of the header).